pages/post/[interviewId].js

```diff
@@ -19,6 +24,10 @@ export default function InterviewArticle() {
       </ImageContainer>
       <TextContainer>
         <Markdown>{article?.text}</Markdown>
+        <ThumbUpContainer>
+          <ThumbUpIcon onClick={() => setLikes(likes + 1)} />
```

Comment:
This should be a proper `<button>` - you can put an `<img>` inside it

Reply:
Yes, sounds better. Will most likely change this in the morning and clean up. Thank you. :)